doc/modules/neighbors.rst

```diff
@@ -464,6 +464,20 @@ leaf nodes.  The level of this switch can be specified with the parameter
 
 ``leaf_size`` is not referenced for brute force queries.
 
+Valid Metrics for Nearest Neighbor Algorithms
+---------------------------------------------
+
+For a list of available metrics, see the documentation of the :class:`DistanceMetric`
+class.
+
+ A list of valid metrics for any of the above algorithms can be obtained by using their
```

Comment:
```suggestion
A list of valid metrics for any of the above algorithms can be obtained by using their
```
Hi @hongshaoyang , I think this extra blank space is the reason of the warning ...

Reply:
Thank you!!